core/src/main/scala/pickling/Pickler.scala

```diff
@@ -26,6 +27,7 @@ trait SPickler[T] {
 
 object SPickler {
   def generate[T]: SPickler[T] = macro Compat.PicklerMacros_impl[T]
+  implicit def spuToSp[T: SPicklerUnpickler]: SPickler[T] = implicitly[SPicklerUnpickler[T]].pickler
```

Comment:
what are the pros/cons of this path vs `SPicklerUnpickler[T] extends SPickler[T] with Unpickler[T]`?